packages/components/src/Components/NotAuthorized/Icon403.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Please don't use this component, it is already exported by PF for us
```JSX
import { LockIcon } from '@patternfly/react-icons`;
```

you can check the full list of icons at https://patternfly-react.surge.sh/icons/